app/tasks/raster_tile_set_assets/raster_tile_set_assets.py

```diff
@@ -67,13 +68,42 @@ async def raster_tile_set_asset(
 
     creation_options = PixETLCreationOptions(**co)
 
+    jobs: List[Job] = list()
     callback: Callback = callback_constructor(asset_id)
 
-    create_raster_tile_set_job: Job = await create_pixetl_job(
-        dataset, version, creation_options, "create_raster_tile_set", callback
+    unify_job: Job | None = None
```

Comment:
I think "Job | None" is more usually expressed as Optional[Job], probably more readable.

Reply:
If you think it's more readable, I'll change it! It's true that it had to be expressed as you say until Python 3.10. This is my first time actually using the new syntax.